docs/event_subscriptions.md

```diff
@@ -10,14 +10,14 @@ As with the default alarms in Guardian, there are default events for some resour
 
 ## Overriding Defaults
 
-Default properites of the events can be overridden through the config YAML using the `EventsSubscription` top level key.
+Default properites of the events can be overridden through the config YAML using the `EventsSubscriptions` top level key.
```

Comment:
@mitchclarkebase2 Not your doing, but would you mind updating properites > properties please